### PR TITLE
Update "About" FlowForge page

### LINKED ISSUE #798: Update About FlowForge page
### Description

The About FlowForge page is out of date. I would like to suggest we rewrite to focus on the FlowForge story.  

Key topics should include:
- How FlowForge got created.
- Mission of FlowForge
- Updated company values
- Investors
- Team
- Careers / Jobs

### Epic/Story

_No response_

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

### PR DESCRIPTION
**Pull request by Yndira-FlowForge**
## Description

Updated the layout and info of the entire page

## Related Issue(s)

Closes #798

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by joepavitt** [comment]
Thoughts on changing from `font-bold` to `font-medium`?

bold:

Screenshot 2023-07-28 at 08 20 51

medium:

Screenshot 2023-07-28 at 08 22 01

**Comment by joepavitt** [comment]
should also add:

```css
align-items: center;
```

to the `.investors .logos` class to vertically align them.

**Comment by joepavitt** [comment]
Other than the above - all good

**Comment by Yndira-FlowForge** [comment]
Hi @joepavitt, thanks for the feedback! I already pushed the changes.

File: src/about.njk [hunk 1]
```diff
@@ -1,56 +1,154 @@
...
+    <div class="w-full pt-12 pb-16 md:pb-10 md:pt-12 flex flex-col items-center">
+        <div class="container text-center max-w-xl md:max-w-4xl px-6">
+            <h1>About FlowForge</h1>
+            <div class="flex flex-col md:flex-row gap-8 items-center mt-10">
+                <div class="md:w-1/2 text-left my-auto">
+                    <p><span class="font-medium">FlowForge</span> was founded in 2021 by <span class="font-medium">Nick
+                            O’Leary, <span class="inline-block">co-creator</span> of Node-RED</span>. The goal of
+                        FlowForge was to create a platform around Node-RED that makes it easier to adopt Node-RED at
+                        scale and <span class="font-medium">integrate into existing enterprise environments</span>.</p>
+                    <p>Today, numerous companies are turning to the FlowForge platform to meet their needs for a <span
```
Comment (iskerrett) [inline src/about.njk:19]:
```suggestion
                    <p>Today, companies are turning to the FlowForge platform to meet their needs for a <span
```

**Review by iskerrett — APPROVED** [review]

**Pull request merged**